src/test/java/uk/gov/justice/digital/delius/StaffRepositoryTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.delius;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Staff;
+import uk.gov.justice.digital.delius.jpa.standard.repository.StaffRepository;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+/*
+ * Uncomment this test and run manually to ensure that the data is loading
+ * and a Staff record can be found by ID.
+ */
+
+@Ignore
```

Comment:
What we doing with this?

Reply:
Its a useful test, though does not assert anything at present. I used it interactively to check that the StaffRepository methods could find staff members by methods, much like a similar offenderRepository test does. I will remove the @Ignore and assert the expected value.